src/lambda/commands/createNewSamApp.ts

```diff
@@ -437,3 +451,43 @@ async function showCompletionNotification(appName: string, configs: string): Pro
         vscode.env.openExternal(vscode.Uri.parse(debugNewSamAppUrl))
     }
 }
+
+/**
+ * Creates a new SAM readme tailored to the IDE and created launch configs
+ * @param readmeLocation Location of new readme file
+ * @param configurations Debug configs to list in readme
+ * @returns True on success, false otherwise
+ */
+export async function writeToolkitReadme(
+    readmeLocation: string,
+    configurations: vscode.DebugConfiguration[] = [],
+    getText: (path: string) => Promise<string> = readFileAsString
+): Promise<boolean> {
+    try {
+        const configString: string = configurations.reduce((acc, cur) => {
+            return `${acc}
```

Comment:
Does this work or does it do weird things?
```js
return `${acc}\n* ${cur.name}`
```

Reply:
Tentatively looks like it works. I'll swap it over and check.